doozerlib/metadata.py

```diff
@@ -437,7 +451,7 @@ def latest_build_list(pattern_suffix):
                     # included in the "-i" doozer argument). We need it to find the pinned NVR
                     # to place in its Dockerfile.
                     # Pinning also informs gen-payload when attempting to assemble a release.
-                    is_nvr = isd.nvr
+                    is_nvr = isd._nvr
```

Comment:
Is this a mistake? Should be just `isd.nvr`.

Reply:
Nice find!